app/Question.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Question extends Model
+{
+	/**
+	*	Attributes to be mass filled.
+	*/
+    protected $fillable = [
+        'user_id', 'organization_id', 'event_id', 'question', 
+        'question_body', 'answer', 'answered_at'
+    ];
+    /**
```

Comment:
concerning number 2, I think having a question main header and a question body would make things look better and more organized. Some volunteers may have relatively long questions or may need to elaborate their questions a little bit more, of course not having a question body will not stop them from doing so and thus would make the event page a little ugly. From another perspective, A user browsing the questions will have an easier time browsing when they can see a brief header about the question instead the bulk of the question itself. Also and with all due respect, I don't think you should immediately shun debatable issues without more elaboration and discussion with the person conceiving said issue first.